test/src/test/java/com/alibaba/nacos/test/naming/RegisterInstance_ITCase.java

```diff
@@ -31,10 +30,16 @@
 import org.springframework.boot.web.server.LocalServerPort;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import java.util.*;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 
-import static com.alibaba.nacos.test.naming.NamingBase.*;
+import static com.alibaba.nacos.test.naming.NamingBase.TEST_IP_4_DOM_1;
```

Comment:
Merge into `import static com.alibaba.nacos.test.naming.NamingBase;`

and make follow constants with `NamingBase.XXX`